src/bindings/python/tests/test_graph/test_custom_op.py

```diff
@@ -85,11 +86,12 @@ def create_add_model():
 class CustomOpWithAttribute(Op):
     class_type_info = DiscreteTypeInfo("CustomOpWithAttribute", "extension")
 
-    def __init__(self, inputs, attrs):
+    def __init__(self, inputs=None, attrs=None):
         super().__init__(self)
-        self._attrs = attrs
-        self.set_arguments(inputs)
-        self.constructor_validate_and_infer_types()
+        if attrs is not None or inputs is not None:
```

Comment:
What happens in case when attributes is not None but inputs is None?

Reply:
actually nothing😅
updated it